### PR TITLE
Package the project and create an CLI app

### PR DESCRIPTION
**Pull request by J535D165**
# Goals 

## Setup & Packaging

``` bash
pip install .
```

- [x] Make package installable

## CLI 
Create an app that allows users to use the tool. 

``` bash
asr my_sr.json --batch_size=10 --query-strategy=lc \
   --pos_labels=110,72,43,2 --neg_labels=22,67,8
```
- [x] CLI with name `asr`
- [x] Make use of modAL package (see #4 ) 
- [x] Good arguments for `asr` 
- [x] Oracle modus
- [x] Interactive modus

## API

``` python
from asr import Review, load_data

data = load_data('data/ptsd.json')

app = Review(data)
app.start()
```

- [x] Make use of modAL package (see #4 ) 
- [x] Python API

## Project cleaning 

- [x] Remove datasets from the project
- [x] Remove all HPC code

### CODE REVIEW

**Comment by J535D165** [comment]
Shall we squash and merge this one? The README is up to date. From now on, I think it is possible to make smaller PRs. @parisa-zahedi @qubixes @daob

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,5 @@
-keras
```
Comment (qubixes) [inline requirements.txt]: requirements.txt seems to be inconsistent with setup.py

Comment (J535D165) [reply requirements.txt]: Fixed in e0ad43d

**Comment by J535D165** [comment]
@parisa-zahedi @qubixes

**Pull request merged**